adapter-venice/src/main/resources/scripts/create-store.sh

```diff
@@ -0,0 +1,24 @@
+#bin/bash
+# This is an utility script to create the store in Venice.
+# Use ./download.sh in order to download the binaries needed to run this script.
+
+set -x -e
+HERE=$(realpath $0)
+cd $HERE
+
+jar=binaries/*admin-tool-all*.jar
+storeName=$1
+url=http://localhost:5555
+clusterName=venice-cluster0
+keySchema=key.avsc
+valueSchema=value.avsc
+
+# create the store
+java -jar $jar --new-store --url $url --cluster $clusterName  --store $storeName --key-schema-file $keySchema --value-schema-file $valueSchema --hybrid-data-replication-policy NON_AGGREGATE
```

Comment:
It seems that `store1` was already present. Is there a better way to start venice with gradlew that will assure no `store1` is present?

Reply:
Unfortuantely for Venice Standalone there is still no "--wipe-data" command.

Reply:
you can wipe everything with this command on the venice root directory
`rm -Rf services/venice-standalone/zookeeperData services/venice-standalone/kafka-logs services/venice-standalone/rocksdb`

I am not adding this note here, I will open a feature request on Venice Standalone